wagtail/admin/templates/wagtailadmin/shared/page_status_tag.html

```diff
@@ -1,18 +1,15 @@
-{% load i18n %}
+{% load i18n wagtailadmin_tags wagtailcore_tags %}
 {% if page.live %}
-    {% with page_url=page.url %}
```

Comment:
@salty-ivy  it was this line that was causing the error - note that in the original code the `page_url` was declared and wrapped in `with`.

I think this probably also caused a bit of a problem with other changes in this file when merging `main` in earlier.

I see the goal here of using the `pageurl`  template tag but will lean towards leaving the previous `with` statement in place.

I will push up the changes to this file and see how we go. Will also do another round of testing.

Reply:
ahhhh i seee, @lb-  thank you so much, for finding the root cause, i wasn't able to find out what's actually causing it, should have looked harder !!!sorry for the trouble.

Reply:
note: please use the `pageurl` tag (`{% pageurl page as page_url %}` as it make use of the request and benefit from cached site parts and more

Reply:
If it's ok @zerolab I'd like to get that in as a seperate PR - unless I've missed something that's not being used now in this file. 

When it was being used it caused some other tests to fail so it probably is worth its own PR.

I'm happy to ensure that gets done once this PR is merged.

Reply:
Works for me! Will look at submitting a separate PR that does it across the admin, not just the one template

Reply:
Issue raised https://github.com/wagtail/wagtail/issues/10125